### PR TITLE
Template: Add answer for legal aspects data owner and modify verbiage

### LINKED ISSUE #268: Update Step 6 Legal aspects data owner question in DAMAP and export templates
**Changes**
1) The last question in the Legal Aspects section is "who is the owner of the data and who controls access to tthe data?"  
**Fix:** change the wording to say "List who will have the rights to control access over which dataset(s)."

2) Currently the answer above does not appear in the templates 
**Fix:** add this statement “The following individual(s) hold rights and control access to the project data:”  and insert the answer from the above question to section 4b (Science Europe), 6.2 (Horizon Europe) and V.1 (FWF). 

3) Currently, if no answer to the question is provided, the phrase "There are no legal restrictions on the processing and disclosure of our data." appears
**Fix:** change the phrase to "“Information regarding rights and control of access to data has yet to be identified.”

4) Adjust the verbiage for "Intellectual property rights and ownership" sections in templates
**Fix:**  change the name of the section to "Intellectual property rights and rights of use" in 4b (Science Europe), 6.2 (Horizon Europe) and V.1 (FWF).

### PR DESCRIPTION
**Pull request by andresTabiTuwien**
## Description

Add to the templates the answer to the question List who will have the rights to control access to which dataset(s) and update the wording in the templates for the Intellectual Property Rights section.

https://github.com/tuwien-csd/damap-frontend/issues/268

#### What does this PR do?

Update the wording in the Intellectual Property section, add to the templates the answer to the question List who will have the rights to control access to which dataset(s).
### Checks

- [ ] Tested with Oracle/PostgreSQL
- [ ] Export updated
- [ ] Documentation added
- [ ] Tests added
- [ ] E2e tests created
- [ ] Successfully ran e2e tests before merge

closes GH-268

### CODE REVIEW

**Comment by andresTabiTuwien** [comment]
[FWFTemplate_review.docx](https://github.com/user-attachments/files/17229739/FWFTemplate_review.docx)
[horizonEuropeTemplate_review.docx](https://github.com/user-attachments/files/17229741/horizonEuropeTemplate_review.docx)
[scienceEuropeTemplate__review.docx](https://github.com/user-attachments/files/17229743/scienceEuropeTemplate__review.docx)

File: src/main/resources/org/damap/base/template/scienceEuropeTemplate.resource
Comment (GeoffreyKarnbach) [inline src/main/resources/org/damap/base/template/scienceEuropeTemplate.resource:1]: Did you forget the "legalRights.owners.no" for this resource file?

Comment (andresTabiTuwien) [reply src/main/resources/org/damap/base/template/scienceEuropeTemplate.resource:1]: Fixed it

**Review by GeoffreyKarnbach — APPROVED** [review]
LGTM!

**Pull request merged**